### DISCUSSION TITLE
How to make the AI write more details.

### DISCUSSION THREAD

**Discussion by sgsdxzy** [oobabooga/text-generation-webui#220] [Q&A]
For most of the time AI generated story contents are very general. They are poor in details, often just and arc of story or directly the result of actions. Like when you prompt _The warrior set out to defeat the demon king_ and AI answers _The demon king is killed_. Sometimes I want the AI to write the details, the steps of every procedure, like where the warrior goes, who he meets, what legendary equipments he gets hands on, and how he trade blows with the demon king. How to make AI more verbose?
Or, if AI gives a general description of events, what can I do to make it elaborate on the chosen detail. For example if AI generates _The warrior stabs the sword into enemy's heart_, and if the AI is ChatGPT, I can ask _describe the sword_ and get a detailed anwser. However most open AI are not finetuned to chat so you cannot ask them to expand some points. It is possible to let AI regenerate a chosen part, but in finer details, and then chose further in the expanded details, etc. How can I prompt it to do so.

**Comment by pbosh** [comment]
The first way to improve your outcome would be to include what you are looking for before you begin the generations. You could cherry pick from your post as a starting point. Sounds simple but this has worked for me many times.
